packages/react/alert-dialog/src/AlertDialog.stories.tsx

```diff
@@ -48,24 +48,121 @@ export const Controlled = () => {
+        <AlertDialogTrigger className={triggerClass}>delete everything</AlertDialogTrigger>
+        <AlertDialogOverlay className={overlayClass} />
+        <AlertDialogContent className={chromaticContentClass}>
+          <AlertDialogTitle className={titleClass}>Title</AlertDialogTitle>
+          <AlertDialogDescription className={descriptionClass}>Description</AlertDialogDescription>
+          <AlertDialogAction className={actionClass}>Confirm</AlertDialogAction>
+          <AlertDialogCancel className={cancelClass}>Cancel</AlertDialogCancel>
+        </AlertDialogContent>
+      </AlertDialog>
+
+      <h2>Open</h2>
+      <AlertDialog open>
+        <AlertDialogTrigger className={triggerClass}>delete everything</AlertDialogTrigger>
+        <AlertDialogOverlay className={overlayClass} style={{ left: '50%', bottom: '50%' }} />
+        <AlertDialogContent className={chromaticContentClass} style={{ top: '25%', left: '75%' }}>
+          <AlertDialogTitle className={titleClass}>Title</AlertDialogTitle>
+          <AlertDialogDescription className={descriptionClass}>Description</AlertDialogDescription>
+          <AlertDialogAction className={actionClass}>Confirm</AlertDialogAction>
+          <AlertDialogCancel className={cancelClass}>Cancel</AlertDialogCancel>
+        </AlertDialogContent>
+      </AlertDialog>
+    </div>
+
+    <div>
+      <h1>Data attribute selectors</h1>
+      <h2>Closed</h2>
+      <AlertDialog>
+        <AlertDialogTrigger className={triggerAttrClass}>delete everything</AlertDialogTrigger>
+        <AlertDialogOverlay className={overlayAttrClass} />
+        <AlertDialogContent className={contentAttrClass}>
+          <AlertDialogTitle className={titleAttrClass}>Title</AlertDialogTitle>
+          <AlertDialogDescription className={descriptionAttrClass}>
+            Description
+          </AlertDialogDescription>
+          <AlertDialogAction className={actionAttrClass}>Confirm</AlertDialogAction>
+          <AlertDialogCancel className={cancelAttrClass}>Cancel</AlertDialogCancel>
+        </AlertDialogContent>
+      </AlertDialog>
+
+      <h2>Open</h2>
+      <AlertDialog defaultOpen>
+        <AlertDialogTrigger className={triggerAttrClass}>delete everything</AlertDialogTrigger>
+        <AlertDialogOverlay className={overlayAttrClass} style={{ top: '50%' }} />
+        <AlertDialogContent className={contentAttrClass} style={{ top: '75%' }}>
+          <AlertDialogTitle className={titleAttrClass}>Title</AlertDialogTitle>
+          <AlertDialogDescription className={descriptionAttrClass}>
+            Description
+          </AlertDialogDescription>
+          <AlertDialogAction className={actionAttrClass}>Confirm</AlertDialogAction>
+          <AlertDialogCancel className={cancelAttrClass}>Cancel</AlertDialogCancel>
+        </AlertDialogContent>
+      </AlertDialog>
+    </div>
+  </div>
+);
+Chromatic.parameters = { chromatic: { disable: false } };
+
+const triggerClass = css({});
```

Comment:
Doesn't look like we need this? But maybe you did it for consistency?

Reply:
Yeah we do this everywhere for consistency.